api/v2/views/reporting.py

```diff
@@ -242,5 +255,11 @@ def list(self, request, *args, **kwargs):
         """
         query_params = self.request.query_params
         if not query_params.items():
-            return failure_response(status.HTTP_400_BAD_REQUEST, "The reporting API should be accessed via the query parameters: ['start_date', 'end_date', 'provider_id']")
-        return super(ReportingViewSet, self).list(request, *args, **kwargs)
+            return failure_response(status.HTTP_400_BAD_REQUEST,
+                                    "The reporting API should be accessed via the query parameters:"
+                                    " ['start_date', 'end_date', 'provider_id']")
+        try:
+            results = super(ReportingViewSet, self).list(request, *args, **kwargs)
+        except ValueError:
```

Comment:
In retrospect there is a better way to do this We could add a try-catch for ValueError in the [`get_filter_query` method](https://github.com/iPlantCollaborativeOpenSource/atmosphere/pull/216/files#diff-f96940a27701dfcc2c1602d0fb5b7cdfR229), then raise a custom exception to catch here.

Or even better is what you suggested - which is to just use DRF's query parameter validation/serialization. :)